files/docker-container-logrotate

```diff
@@ -1,9 +1,11 @@
 /var/lib/docker/containers/*/*.log {
   rotate 0
-  daily
+  hourly
```

Comment:
Had an offline chat with Dhawal, adding here for reference for others
Timer will run with whatever cadence we run it. So at each hourly run of the timer, whether to run logrotate service or not, that depends upon the config. It will run in two cases with our config, one is based on last run datetime, if its past 1hr, it will run and second is based on size. So if the log size is above 10mb, it will run the logrotate service